apps/chat/channels.py

```diff
@@ -201,16 +207,26 @@ def participant_user(self):
         if self.experiment_session:
             return self.experiment_session.participant.user
 
-    def send_voice_to_user(self, synthetic_voice: SynthesizedAudio, attached_files: list[File] = None):
+    def send_voice_to_user(self, synthetic_voice: SynthesizedAudio):
         raise NotImplementedError(
             "Voice replies are supported but the method reply (`send_voice_to_user`) is not implemented"
         )
 
     @abstractmethod
-    def send_text_to_user(self, text: str, attached_files: list[File] = None):
+    def send_text_to_user(self, text: str):
         """Channel specific way of sending text back to the user"""
         raise NotImplementedError()
 
+    def send_file_to_user(self, files: list[File]):
+        """
+        Sends the file to the user. This is a channel specific way of sending files.
+        The default implementation does nothing.
+        """
+        pass
+
+    def _can_send_file(self, file: File) -> bool:
+        return False
```

Comment:
how is this different from `supports_multimedia`?

Reply:
`supports_multimedia` indicated that we can send files to the channel, but since the channel impose restrictions on file size and types, we need to check each file individually to see if they can be send to the channel. This is where `_can_send_file comes in